app/views/email_authentications/invitations/_access_tree.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="card access-tree">
```

Comment:
Pulling this out in the partial will come in handy when we use the same general thing for edit and view both.